libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -367,8 +367,29 @@ void ReferencesResolver::endVisit(VariableDeclaration const& _variable)
 				}
 				else if (varLoc == Location::Default)
 				{
-					if (_variable.isCallableParameter())
-						typeLoc = DataLocation::Memory;
+					if (_variable.isCallableParameter()) {
+                        typeLoc = DataLocation::Memory;
```

Comment:
Please indent using tabs only.
Please put `{` on a line of its own.

Coding style: https://github.com/ethereum/solidity/blob/develop/CODING_STYLE.md

Reply:
My apologies on this! Should be fixed now.